requirements/base.in

```diff
@@ -6,3 +6,5 @@ mypy
 pycryptodome>=3.17.0
 pyyaml>=6.0
 typing-extensions>=4.4.0
+importlib-metadata>=7.0.1
+importlib-resources>=6.1.1
```

Comment:
You should recompile requirement files with `make compile-requirements`.

Reply:
Alright, we'll do that as well. Thank you for letting us know.